Translations/pt_BR.po

```diff
@@ -1414,19 +1408,19 @@ msgstr "Tecla de atalho para habilidade ou feitiço."
 
 #: Source/diablo.cpp:1622 Source/diablo.cpp:1995
 msgid "Use health potion"
-msgstr ""
+msgstr "Usar poção de vida"
 
 #: Source/diablo.cpp:1623 Source/diablo.cpp:1996
 msgid "Use health potions from belt."
-msgstr ""
+msgstr "Usar poção de vida do cinto."
```

Comment:
Original is plural:
"Usar poções de vida do cinto."

Reply:
Oops, I didn't notice.